doc/modules/ROOT/pages/tutorials/import-sample-export-gnn.adoc

```diff
@@ -11,6 +11,11 @@ This Jupyter notebook is hosted
 https://github.com/neo4j/graph-data-science-client/blob/main/examples/import-sample-export-gnn.ipynb[here]
 in the Neo4j Graph Data Science Client Github repository.
 
+The notebook is featured in the tutorial
```

Comment:
I think this would be more eye-catching if they know its a video presentation.
I'd go with "For a video presentation of this notebook, see the talk [...]"

Reply:
good point :+1: